### PR TITLE
Sample SQL for a dashboard: counts of texters, sent, responses, total texts, opt outs

### PR DESCRIPTION
**Pull request by joemcl**
added SQL to generate this - 
`counts of texters, sent, responses, total texts, opt outs and estimated cost for all current campaigns
`

### CODE REVIEW

File: docs/TEXTER_ACTIVITY_QUERIES.md [hunk 1]
```diff
@@ -99,10 +95,53 @@ GROUP BY
...
+`SELECT
+    c.id AS campaign_id,
+    c.title AS campaign,
+    texters::int,
+    texts_sent::int,
+    replies::int,
+    total_texts::int,
+    opt_outs::int,
+    /* cost per texter = $1 / 30 days (est) */
+    ROUND(((texters / 30)  + ((total_texts * .015)*1.15)), 2) AS costid
```
Comment (shakalee14) [inline docs/TEXTER_ACTIVITY_QUERIES.md]: hey @joemcl - where did you get this `.015` cost from? can you make a note that this is factoring text messages to be 2 segments (so 2 x .0075)? thanks!

Comment (joemcl) [reply docs/TEXTER_ACTIVITY_QUERIES.md]: yup will do @shakalee14

**Comment by joemcl** [comment]
@shakalee14 done, updated...

**Review by shakalee14 — APPROVED** [review]

**Comment by joemcl** [comment]
thanks @shakalee14 !

**Pull request merged**